dir.props

```diff
@@ -9,6 +9,10 @@
     <!-- Temp change to make OS X build behave as a Linux build -->
     <OsEnvironment Condition="'$(OsEnvironment)'=='' AND '$(OS)'=='OSX'">Unix</OsEnvironment>
     <OsEnvironment Condition="'$(OsEnvironment)'==''">$(OS)</OsEnvironment>
+
+    <!-- ToDo: Eliminate the need for this property -->
+    <ContainsPackageReferences>true</ContainsPackageReferences>
```

Comment:
@mellinoe should we just remove this property and force the move as using project.json file doesn't seem to work anyway.